src/components/SoraCard/steps/TermsAndConditions.vue

```diff
@@ -24,8 +24,8 @@
     <s-button type="primary" class="sora-card__btn s-typography-button--large" @click="handleConfirmToS">
       <span class="text">ACCEPT & CONTINUE</span>
     </s-button>
-    <tos-dialog :visible.sync="showTermsAndConditionsDialog" :srcLink="TosExternalLinks.Terms" />
-    <tos-dialog :visible.sync="showPrivacyPolicyDialog" :srcLink="TosExternalLinks.Privacy" />
+    <tos-dialog :visible.sync="showTermsAndConditionsDialog" :srcLink="termsLink" />
+    <tos-dialog :visible.sync="showPrivacyPolicyDialog" :srcLink="privacyLink" />
```

Comment:
could we use only one instance of the dialog component?

Reply:
causing some caching issues regarding provided links
:key does not help as it brings another bug